website/docs/learning-and-resources/how-to-guides/how-to-use-data-from-async-functions-in-sync-fields.md

```diff
@@ -0,0 +1,86 @@
+---
+description:
+  Learn how to use data from asynchronous functions in synchronous fields in Appsmith to build efficient apps.
+---
+
+# How to Use Data from Async Functions in Sync Fields
+In Appsmith, it's common to use [async/await](/core-concepts/writing-code/javascript-promises#asyncawait)  functions to handle asynchronous operations. For example, fetching data from an API or a database. Using async functions in sync fields can be more complex when binding data to a widget.
+
+ <VideoEmbed host="youtube" videoId="yn_8gs5w04g" title="Using data from Async function in Synchronous Field" caption="Using data from Async function in Synchronous Field"/> 
+
+This guide helps you understand using async functions in sync fields. It focuses on a practical use case and code examples to fetch data by triggering APIs,  JS Object, or a query.
+
+## Display user information on a page
+
+Consider an example where you fetch the user information and show it in a Table widget. You can fetch the user information using an API, a database query, or a JS Object function.
+
+### Fetch user data using an API
```

Comment:
I think I had called this out in the original PR when we added this information. The main title of the page is talking about using async **functions** in sync fields. meaning JS functions. we are not talking about db/api queries. I think if that is the case, we should rethink this page and add the all of this information but in a different way on the original Workflows doc.

Can do this over a call as it might easier to understand